config/task-config.js

```diff
@@ -2,10 +2,11 @@ module.exports = {
   html        : true,
   images      : true,
   fonts       : true,
-  static      : true,
-  svgSprite   : true,
-  ghPages     : true,
+  static      : false,
+  svgSprite   : false,
+  ghPages     : false,
   stylesheets : true,
+  icons       : false,
```

Comment:
Hmmm, I don't think blendid has an icons task so you could take out this line in the task config.